src/main/features/response/routes/your-defence.ts

```diff
@@ -40,6 +41,6 @@ export default express.Router()
 
         await new DraftService().save(res.locals.user.responseDraft, res.locals.user.bearerToken)
 
-        res.redirect(Paths.freeMediationPage.evaluateUri({ externalId: user.claim.externalId }))
+        res.redirect(Paths.taskListPage.evaluateUri({ externalId: user.claim.externalId }))
```

Comment:
has the flow changed? It will require updating int tests

Reply:
Yes, that was the heart of the story :)

Reply:
The unit test were updated to match this change